### DISCUSSION TITLE
Onnxruntime C++ resize/reshape/squeeze

### DISCUSSION THREAD

**Discussion by EmreOzkose** [microsoft/onnxruntime#13163] [Other Q&A]
Hi,

Is there a way to squeeze Ort::Value?

**Comment by ytaous** [comment]
not quite follow your question, here is UTs on Squeeze for example - https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/test/providers/cpu/tensor/squeeze_op_test.cc
